substrate/frame/revive/uapi/src/lib.rs

```diff
@@ -22,9 +22,36 @@
 mod flags;
 pub use flags::*;
 mod host;
+mod macros;
 
 pub use host::{HostFn, HostFnImpl};
 
+#[cfg(feature = "fixtures")]
+#[panic_handler]
+fn panic(_info: &core::panic::PanicInfo) -> ! {
+	// Safety: The unimp instruction is guaranteed to trap
+	unsafe {
+		core::arch::asm!("unimp");
+		core::hint::unreachable_unchecked();
+	}
+}
```

Comment:
This doesn't make sense. The feature should enable building the fixtures. Not adding a panic handler to the crate itself.  Adding a panic handler makes it unusable for other use cases. I thought in order to get rid of the common crate just use `include_file!` with a file that is inside the fixtures crate.

Reply:
> I thought in order to get rid of the common crate just use include_file!

We could do that as well, then no need for this ff, I just moved it here so that we can add the panic handler to fixtures contracts, but yeah include_file would work as well